Unable to compile master branch e4224d2b8812b701ba8d6f3b9078f13c0e144963

@hboon When building project the following errors are reported:
```
/Users/jerome/Documents/alpha-wallet-ios/AlphaWallet/Core/Views/WebImageView.swift:113:63: error: 'value' is inaccessible due to 'internal' protection level
            if let data = try? ImageCache.default.diskStorage.value(forKey: url.absoluteString), let svgString = data.flatMap({ String(data: $0, encoding: .utf8) }) {
                                                              ^~~~~
/Users/jerome/Documents/alpha-wallet-ios/Pods/Kingfisher/Sources/Cache/DiskStorage.swift:160:14: note: 'value(forKey:extendingExpiration:)' declared here
        func value(forKey key: String, extendingExpiration: ExpirationExtending = .cacheTime) throws -> T? {
             ^
/Users/jerome/Documents/alpha-wallet-ios/AlphaWallet/Core/Views/WebImageView.swift:134:61: error: 'store' is inaccessible due to 'internal' protection level
                        try? ImageCache.default.diskStorage.store(value: data, forKey: url.absoluteString)
                                                            ^~~~~
/Users/jerome/Documents/alpha-wallet-ios/Pods/Kingfisher/Sources/Cache/DiskStorage.swift:110:14: note: 'store(value:forKey:expiration:)' declared here
        func store(
```
I've checked out the branch on it's own and tried to build it.

--- a/Podfile
+++ b/Podfile
@@ -16,7 +16,7 @@ target 'AlphaWallet' do
   pod 'RealmSwift', '5.5.1'
   pod 'Moya', '~> 10.0.1'
   pod 'CryptoSwift', '~> 1.4'
-  pod 'Kingfisher', '5.15.7'
+  pod 'Kingfisher', '~> 7.0'
   pod 'AlphaWalletWeb3Provider', :git=>'https://github.com/AlphaWallet/AlphaWallet-web3-provider', :commit => '9a4496d02b7ddb2f6307fd0510d8d7c9fcef9870'
   pod 'TrezorCrypto', :git=>'https://github.com/AlphaWallet/trezor-crypto-ios.git', :commit => '50c16ba5527e269bbc838e80aee5bac0fe304cc7'
   pod 'TrustKeystore', :git => 'https://github.com/AlphaWallet/latest-keystore-snapshot', :commit => 'c0bdc4f6ffc117b103e19d17b83109d4f5a0e764'

